app/helpers/root_helper.rb

```diff
@@ -2,21 +2,40 @@ module RootHelper
   MAX_CHAR = 40
 
   def show_popular_links
-    links = map_link_details
+    links = map_link_details_popular
     render partial: "link_card", collection: links, as: :card_link
   end
 
-  def map_link_details
+  def show_recent_links
+    links = map_link_details_recent
+    render partial: "link_card", collection: links, as: :card_link
+  end
+
+  def map_link_details_popular
     @popular.map do |link|
-      full_url = link.full_url
-      click_count = link.count.to_s << (link.count == 1 ? " click" : " clicks")
-      get_object(
-        click_count, full_url, get_compact_form(full_url),
-        generate_link(link)
-      )
+      click_count =
+        link.count.to_s <<
+        (link.count == 1 ? " click" : " clicks")
+      map_helper(click_count, link)
+    end
+  end
+
+  def map_link_details_recent
+    @recent.map do |link|
+      time_distance =
+        distance_of_time_in_words(link.created_at, Time.now) << " ago"
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.